model.js

```diff
@@ -69,3 +69,23 @@ exports.selectReviews = () => {
       return result.rows;
     });
 };
+
+exports.selectReviewComments = (reviewId) => {
+  return connection
+    .query(
+      `SELECT comments.* FROM reviews
+      LEFT JOIN comments ON comments.review_id = reviews.review_id 
+      WHERE reviews.review_id = $1`,
+      [reviewId]
+    )
+    .then((result) => {
+      if (result.rows.length === 0) {
+        return Promise.reject({
+          msg: "Review ID not found",
```

Comment:
A result of 0 rows doesn't always mean that the review does not exist. There will be cases where the review exists but has no comments yet, such as when it's first made. Add some tests for these cases and make sure your endpoint behaves correctly.